### PR TITLE
Update comment

### PR DESCRIPTION
**Pull request by aiucr14**

### CODE REVIEW

**Comment by bryndi** [comment]
ok

**Comment by ericalloyd** [comment]
General high-level comment

File: _slides/04-aiucr14.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Hello !
```
Comment (ericalloyd) [inline _slides/04-aiucr14.md]: LGTM!
👍🏿 🚢 🚀 ✅

Comment (ericalloyd) [inline _slides/04-aiucr14.md]:
```suggestion
Hello!
```

**Review by ericalloyd — CHANGES_REQUESTED** [review]
This looks good, but requesting 1 change

**Review by ericalloyd — APPROVED** [review]

**Pull request merged**